File: Ruby/6/example6.rb
#!/usr/bin/env ruby2

outfile = 'out.txt'

text = "aldsfkjlsadfkj\nalsdfkjalsdfjk\nalsdkjflasdfjk\nlasdkjlsadkjf\n"

puts "Writing to outfile ..."
File.write outfile, text

puts "Reading from outfile ..."

File.open(outfile).each do |line|
    puts line
end

puts 'Adding more text to outfile ...'
more_text = "alsdkjfalfdkj\nalsdfjldfkj\n"

File.write outfile, more_text, mode: 'a'

puts 'Reading entire file ...'
puts File.read outfile

File: Python/6/example6.py
#!/usr/bin/env python

outfile = 'out.txt'

text = 'aldsfkjlsadfkj\nalsdfkjalsdfjk\nalsdkjflasdfjk\nlasdkjlsadkjf\n'

print('Writing to outfile ...')
open(outfile, 'w').write(text)

print('Reading from outfile ...')
for line in open(outfile, 'r').readlines():
    print(line)

print('Adding more text to outfile ...')
more_text = 'alsdkjfalfdkj\nalsdfjldfkj\n'

open(outfile, 'a').write(more_text)

print('Opening outfile to read the entire text ...')
out = open(outfile, 'r').read()

print(out)